firmware/arduino/chessboard/movementFunctions.ino

```diff
@@ -38,11 +39,12 @@ bool moveDirect(int startCol, int startRow, int endCol, int endRow)
   //    returns true before anything is done.
   // 4. Now we end up in the first recursive call and simply move to the start point.
   // 5. After being moved to the start point, we perform the move from the initial call.
-  if (!moveDirect(currCol, currRow, startCol, startRow))
+  if (!moveDirect(currCol, currRow, startCol, startRow, false))
```

Comment:
Please add comment explaining that since we're aligning to the start position, we want the electromagnet to be off.